ibm_catalog.json

```diff
@@ -40,7 +40,7 @@
           },
           {
             "description": "Configures Activity Tracker routing",
-            "title": "Configures an Activity Tracker route with a COS bucket target."
+            "title": "Configures an Activity Tracker route with a COS bucket and log analysis target."
```

Comment:
As per this title - we are configuring route for both by default. Is it so?
Accordingly it needs a change.

Reply:
done